src/boundingboxes.jl

```diff
@@ -46,6 +46,15 @@ function boundingbox(s::Sphere{Dim,T}) where {Dim,T}
   Box(c - r⃗, c + r⃗)
 end
 
+function boundingbox(p::ParaboloidSurface{T}) where {T}
+    v = p(T(0), T(0))
+    r = radius(p)
+    f = focallength(p)
+
+    _, cy, _ = coordinates(v)
+    Box(v + Vec(-r, -r, T(0)), v + Vec(r, r, T(r^2 / (4f))))
```

Comment:
```suggestion
    Box(v + Vec(-r, -r, T(0)), v + Vec(r, r, (r^2 / (4f)))
```

Reply:
I'm a bit puzzled here… Since `r` and `f` are already of type `T`, isn't `r^2 / 4r` automatically of type `T`?

```julia
julia> r = Float32(1.0)
1.0f0

julia> f = Float32(4.0)
4.0f0

julia> fn(r, f) = r^2 / (4f)
fn (generic function with 1 method)

julia> fn(r, f)
0.0625f0

julia> typeof(fn(r, f))
Float32

julia> @code_typed fn(r, f)
CodeInfo(
1 ─ %1 = Base.mul_float(r, r)::Float32
│   %2 = Base.mul_float(4.0f0, f)::Float32
│   %3 = Base.div_float(%1, %2)::Float32
└──      return %3
) => Float32
```

Or do you encourage types to be explicit in any case in the Meshes.jl codebase?

Reply:
Exactly, if the inputs are of type T then the expression will be of type T if the literals are integer. We are assuming that T <: Real or a unitful type on top of Real.